currency_rate_update_xe/__manifest__.py

```diff
@@ -3,7 +3,7 @@
 
 {
     "name": "Currency Rate Update: XE.com",
-    "version": "15.0.1.0.0",
+    "version": "16.0.1.0.0",
```

Comment:
This change (and the corresponding README modifications) should be done on the migration commit, not in the pre-commit auto-fixes.

Reply:
Fixed